index.md

```diff
@@ -66,25 +66,21 @@ Not sure where to start? Check out [our resources](#resources).
 
 _For those thinking: “We don’t have anything yet, where do we start?”_
 
-**Model policies and strategies:**
-* [Resources to Help Address Sexual Harassment in Venture Ecosystem](https://nvca.org/pressreleases/nvca-unveils-resources-help-address-sexual-harassment-venture-ecosystem/) (NVCA), which includes (1) sample H.R. policies for addressing harassment & discrimination; (2) sample H.R. best practices for addressing harassment & discrimination; and (3) a sample Code of Conduct
-* [A curated directory of resources, information, and initiative by the Venture Inclusion Network](https://www.ventureinclusion.com/), which includes a list of experts and services, initiatives in the industry, sample policies, best practices and more (Venture Inclusion Network)
+**Templates and Example Policies:**
+* NVCA recently published [resources to help address sexual harassment in the venture ecosystem](https://nvca.org/pressreleases/nvca-unveils-resources-help-address-sexual-harassment-venture-ecosystem/), which includes (1) sample H.R. policies for addressing harassment & discrimination; (2) sample H.R. best practices for addressing harassment & discrimination; and (3) a sample Code of Conduct
+* [Project Include](http://projectinclude.org/), a community for building meaningful, enduring diversity and inclusion into tech companies, has a number of templates available to use
+* [Venture Inclusion Network](https://www.ventureinclusion.com/), hosts a curated directory of experts and services, initiatives in the industry, sample policies, best practices and more
```

Comment:
remove the comma after Venture Inclusion Network

Reply:
+1